Bottom: 83cc9188c6c0ec02270462742d90dc21b11adea9
Top:    d79d2bea76a710e50966923fbe7ce6c99e73ac84
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:50:02 +0100

Refresh of hid-gtk-layer-selector-rename-

---

diff --git a/src/hid/gtk/ghid-layer-selector.h b/src/hid/gtk/ghid-layer-selector.h
index 09fa182..ff67800 100644
--- a/src/hid/gtk/ghid-layer-selector.h
+++ b/src/hid/gtk/ghid-layer-selector.h
@@ -24,7 +24,7 @@ void ghid_layer_selector_add_layer (GHidLayerSelector *ls,
                                     const gchar *name,
                                     const gchar *color_string,
                                     gboolean visible,
-                                    gboolean activatable);
+                                    gboolean selectable);
 
 gint ghid_layer_selector_install_pick_items (GHidLayerSelector *ls,
                                              GtkMenuShell *shell, gint pos);
